FileManager.php

```diff
@@ -368,6 +370,32 @@ private function unsafeDeduplicate(string $filename): string
         return $candidate;
     }
 
+    /**
+     * Use finder to extract files list from a directory
+     * @param $uri is the directory uri
```

Comment:
Blan like before `@param`, parameters must be typed, `@param string $uri`.

Reply:
done